pkgs/development/python-modules/jinja2-strcase/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  freezegun,
+  jinja2,
+  pytestCheckHook,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "jinja2-strcase";
+  version = "0.0.2";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-2Qw3971A00WqzI94sIf2bmxapMloqyOnkVc/z3VsM3k=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+
+  propagatedBuildInputs = [ jinja2 ];
+
+  nativeCheckInputs = [
+    freezegun
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "jinja2_strcase" ];
+
+  doCheck = false;
```

Comment:
Why do you disable tests?

Reply:
With `doCheck = true`, I get this output:
```
@nix { "action": "setPhase", "phase": "pytestCheckPhase" }
Running phase: pytestCheckPhase
Executing pytestCheckPhase
============================= test session starts ==============================
platform linux -- Python 3.12.4, pytest-8.2.2, pluggy-1.5.0
rootdir: /build/jinja2-strcase-0.0.2
collecting ... ^Mcollected 0 items

============================ no tests ran in 0.00s =============================
/nix/store/1r32fki5z5ivgaikgh8pq0r6vz3b0jpj-stdenv-linux/setup: line 1655: pop_var_context: head of shell_variables not a function context
```
 This didn't seem like a failure, so I disabled the tests to avoid spurious output. Was that a bad call?

Reply:
OK, the upstream doesn't have tests. Could you leave a comment that there is no test?

Reply:
> Could you leave a comment that there is no test?

I edited the PR description to that effect.